catalyst/dl/callbacks/mixup.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+from catalyst.dl.core.state import RunnerState
+from catalyst.dl.callbacks import CriterionCallback
+import numpy as np
+import torch
+
+
+class MixupCallback(CriterionCallback):
+    def __init__(self, fields: List[str] = ('features',), alpha=1.0, train_only=True, **kwargs):
+        assert alpha >= 0, 'alpha must be>=0'
+
+        super(MixupCallback, self).__init__(**kwargs)
+
+        self.train_only = train_only
+        self.fields = fields
+        self.alpha = alpha
+        self.lam = 1
+        self.index = None
+
+    def on_batch_start(self, state: RunnerState):
+        if self.train_only and state.loader_name != 'train':
```

Comment:
it better to use `state.loader_name.startswith('train')`

PS. btw, please use " to follow lib code standarts

Reply:
Fixed. What is the reason to use double quotes instead of a single one? 
To type " we should use Shift on the most keyboards. Do you have one that allows doing type in one press ?)